### PR TITLE
Let this guide display both Groovy and Kotlin DSL samples

### PR DESCRIPTION
**Pull request by eskatos**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -12,8 +12,9 @@ guide {
 
 asciidoctor {
     inputs.dir 'src/example'
-    attributes exampledir : file('src/example')
-    attributes 'gradle-outdir' : "${gradleRunner.reportsDir.absolutePath}"
+    attributes 'groovy-example-dir': file('src/example/groovy-dsl')
+    attributes 'kotlin-example-dir': file('src/example/kotlin-dsl')
+    attributes 'gradle-outdir': "${gradleRunner.reportsDir.absolutePath}"
```
Comment (bamboo) [inline build.gradle]: ❓ Is there a reason to prefer `...: "${*.absolutePath}"` over `...: *.absolutePath`?

File: src/example/kotlin-dsl/docs/build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import org.asciidoctor.gradle.AsciidoctorTask
+
+plugins {
+    id("org.asciidoctor.convert")       // <1>
+}
+
+val asciidoctor = tasks.getByName<AsciidoctorTask>("asciidoctor") {
+    sources(delegateClosureOf<PatternSet> {
```
Comment (bamboo) [inline src/example/kotlin-dsl/docs/build.gradle.kts]: ⭕️ We should file an issue on asciidoctor's repository about using `Action<PatternSet>` here instead.

File: src/example/kotlin-dsl/greeter/build.gradle.kts [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+// tag::link-docs[]
+tasks.getByName<Zip>("distZip") {
+    from(project(":docs").tasks["asciidoctor"]) { // <1>
+        into("${project.name}-$version")
+    }
+}
+tasks.getByName<Tar>("distTar") {
+    from(project(":docs").tasks["asciidoctor"]) {
+        into("${project.name}-${version}")
```
Comment (bamboo) [inline src/example/kotlin-dsl/greeter/build.gradle.kts]: :x: `into("${project.name}-$version")`

File: build.gradle (hunk 1 above)
Comment (eskatos) [reply build.gradle]: No reason, I just left it as it was, the sole change here is a removed space before `:`

File: src/example/kotlin-dsl/docs/build.gradle.kts (hunk 2 above)
Comment (eskatos) [reply src/example/kotlin-dsl/docs/build.gradle.kts]: It already exists: https://github.com/asciidoctor/asciidoctor-gradle-plugin/issues/236

File: src/example/kotlin-dsl/greeter/build.gradle.kts (hunk 3 above)
Comment (eskatos) [reply src/example/kotlin-dsl/greeter/build.gradle.kts]: Good catch, fixed!

**Pull request merged**